set-web/src/main/java/eu/europeana/set/web/service/controller/jsonld/WebUserSetRest.java

```diff
@@ -200,7 +207,8 @@
 		userSet = getUserSetService().fetchItems(userSet, sort, sortOrder, pageNr, pageSize, profile);
 	    }
 
-	    String userSetJsonLdStr = serializeUserSet(profile, userSet);
+	    UserSet resUserSet = setTotalField(profile, userSet);
```

Comment:
move to applyProfile method as indicated in previous comment

Reply:
in all similar methods, functionality moved to applyProfile method